### PR TITLE
Branch update with -U

### PR DESCRIPTION
**Pull request by Henryws**
Pacstall can update both user and branch
`sudo pacstall -U wizard-28 develop` for example and if no input is specified, it will default to `pacstall` and `master`

### CODE REVIEW

**Comment by Henryws** [comment]
Don't merge yet

**Review by wizard-28 — CHANGES_REQUESTED** [review]
Also fix the CodeFactor issue

**Comment by Henryws** [comment]
Codefactor failed b/c it thinks that quoting something unquotes it which is what I want to do so it's good to go

File: pacstall [hunk 1]
```diff
@@ -290,15 +290,23 @@ while [[ "$1" =~ ^- && ! "$1" == "--" ]]; do
       fi
       ;;
     -U|--update)
+      USER="$2"
+      BRANCH="$3"
+      if [[ -n "$BRANCH" ]]; then
+          BRANCH=master
```
[the remark this answers is not in the document]
Comment (Henryws) [reply pacstall]: done

**Review by wizard-28 — APPROVED** [review]
Okay then good to go!

**Comment by wizard-28** [comment]
Why is the branch restored?

**Comment by Henryws** [comment]
Oops

**Pull request merged**